### PR TITLE
add root build info to interprojectalert

### PR DESCRIPTION
**Pull request by zdhickman**
## Unexpanded
screen shot 2016-07-07 at 2 53 58 pm

---

## Expanded
screen shot 2016-07-07 at 2 54 04 pm

### CODE REVIEW

**Comment by jonathanwgoodwin** [comment]
@gchomatas

File: BlazarUI/app/scripts/components/repo-build/InterProjectAlert.jsx [hunk 1]
```diff
@@ -162,16 +178,16 @@ class InterProjectAlert extends Component {
       return null;
     }
 
-    const {upstreamRepoBuilds, downstreamRepoBuilds, interProjectBuildId} = upAndDownstreamModules;
+    const {upstreamRepoBuilds, downstreamRepoBuilds, rootRepoBuilds, interProjectBuildId} = upAndDownstreamModules;
 
-    if (Object.keys(upstreamRepoBuilds).length === 0 && Object.keys(downstreamRepoBuilds).length === 0) {
+    if (!(Object.keys(upstreamRepoBuilds).length + Object.keys(downstreamRepoBuilds).length  + Object.keys(rootRepoBuilds).length)) {
```
Comment (andyhuang91) [inline BlazarUI/app/scripts/components/repo-build/InterProjectAlert.jsx]: I think http://underscorejs.org/#isEmpty represents a little more succinctly what you're trying to do.

**Comment by andyhuang91** [comment]
JS looks okay.

**Comment by gchomatas** [comment]
I don't feel that is a good idea to mix the top-level (root) build(s) within the immediate upstream builds. Even adding the extra info in parenthesis to designate the root build it is confusing.
I would imaging something like that:

Top level (or root) Builds that triggered this build: 
  - build 1
  - buil 2

Immediate Upstream Builds:
 - build 3
 - build 4

Immediate Downstream Builds:
  - build 5
  - build 6

The wording in the titles is just indicative and to make clear what I mean, we can use any other more appropriate / more terse wording 

/cc @markhazlewood

**Comment by zdhickman** [comment]
^ Updated

**Comment by zdhickman** [comment]
Update with new styling

**Comment by markhazlewood** [comment]
@gchomatas I was looking at a different layout/design to better highlight the root in the IPB details that might address your concerns:

image

In the case where you are ON the repo build page that triggered the IPB, the text would just read

`ModuleName [buildNumber]`

Rather than

`Repo::ModuleName [buildNumber]`

**Comment by markhazlewood** [comment]
🚸

**Pull request merged**